packages/dbos-sqs/index.ts

```diff
@@ -133,7 +135,13 @@ class SQSReceiver implements DBOSEventReceiver
         for (const registeredOperation of regops) {
             const cro = registeredOperation.classConfig as SQSReceiverClassDefaults;
             const mro = registeredOperation.methodConfig as SQSReceiverMethodSpecifics;
-            const url = cro.config?.queueURL ?? mro.config?.queueURL;
+            const url =
+                cro.config?.queueUrl
+                ?? cro.config?.queueURL
+                ?? mro.config?.queueUrl
+                ?? mro.config?.queueURL;
+            console.log(`${process.env['SQS_QUEUE_URL']} / ${url}`);
```

Comment:
```suggestion
            console.log(`${process.env['SQS_QUEUE_URL']} / ${url}`);
```
I assume this is a debug leftover? No problem using the executor logger otherwise if that's useful information in `DEBUG` level.

Reply:
Sry